.github/workflows/run.yml

```diff
@@ -174,6 +177,7 @@ jobs:
           echo "::set-output name=outputDir::${{ runner.temp }}"
           echo "::set-output name=dockerEnvFile::$dockerEnvFile"
       - name: 'Run tests using ${{ needs.condition.outputs.infra }}'
+        continue-on-error: ${{ matrix.besu-network == 'true' }}
```

Comment:
Why do we need this flag here? Ideally we should see the step with a fail tick rather than with a success one.

Reply:
at the moment, it will always fail as Besu doesn't pass yet. I wanted to add it so I can see the results on a new version of Besu, but maybe I can simply remove it for now and only add it when ready.

Reply:
@nmvalera please check here. I added the Besu tests but for now is ignored as Besu is still not completely sync, but I will enable this flag when the fixes are merged and publish.